src/index.js

```diff
@@ -69,4 +79,45 @@ function clickLog(event) {
     domUpdates.populateCards(user.searchApprovedTrips(event.target.classList.value))
 }
 
+function validateForm() {
+    let validated = false;
```

Comment:
```suggestion
    let validated = true;
```

This way we can keep all of our `if` statements and get rid of the else's.
We can assume it's validated, unless we have a reason to think it's not. 
This would let you delete ~6 lines of code.

Reply:
💯